Source/Details/ASImageProtocols.h

```diff
@@ -209,7 +209,7 @@ withDownloadIdentifier:(id)downloadIdentifier;
 /**
  @abstract Should be called when playback is ready.
  */
-@property (nonatomic, strong, readwrite) dispatch_block_t playbackReadyCallback;
```

Comment:
@maicki @garrettmoon  Great find! Any thoughts on why not put copy here explicitly? That would seem to match the style of most of the other changes; if anything, perhaps removing the readwrite specifier is an option too.

The copy semantics as a default are a bit less clear (since it's a special case) than the readwrite semantics (since it's always the case)